### PR TITLE
Add CCArchiver XML definition

### PR DESCRIPTION
**Pull request by srp3rd**

### CODE REVIEW

**Comment by rbovill** [comment]
In order for the tests to pass, one more change must be made.  Please add CCArchiver to the subsystems list in python/lsst/ts/xml/testutils.py.

**Comment by srp3rd** [comment]
Added CCArchiver to python/lsst/ts/xml/testutils.py

File: sal_interfaces/CCArchiver/CCArchiver_Commands.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <EFDB_Topic>CCArchiver_command_resetFromFault</EFDB_Topic>
+    <Alias>resetFromFault</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>ignored</Description>
```
Comment (mareuter) [inline sal_interfaces/CCArchiver/CCArchiver_Commands.xml]: The correct description for an ignored parameter is “Attribute required by the API, but is unused.”

File: sal_interfaces/CCArchiver/CCArchiver_Telemetry.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://project.lsst.org/ts/sal_objects/schema/SALTelemetrySet.xsd">
+<SALTelemetry>
+    <Subsystem>CCArchiver</Subsystem>
+    <Version>3.8</Version>
+    <Author></Author>
+    <EFDB_Topic>CCArchiver_sequencerHeartbeat</EFDB_Topic>
+    <item>
+        <EFDB_Name>heartbeat</EFDB_Name>
+        <Description>heartbeat from OCS</Description>
```
Comment (mareuter) [inline sal_interfaces/CCArchiver/CCArchiver_Telemetry.xml]: I'm confused by this description. Are you sending a different heartbeat compared to the SALOBJ one?

File: sal_interfaces/CCArchiver/CCArchiver_Commands.xml (hunk 1 above)
Comment (srp3rd) [reply sal_interfaces/CCArchiver/CCArchiver_Commands.xml]: This was replicated from the ATArchiver.  I'll update them from "ignored" to "Attributed required by the API, but is unused."

File: sal_interfaces/CCArchiver/CCArchiver_Telemetry.xml (hunk 2 above)
Comment (srp3rd) [reply sal_interfaces/CCArchiver/CCArchiver_Telemetry.xml]: This was replicated from the ATArchiver, and was in place before I started.  I had presumed this was needed by SALOBJ, but if it's not, I can remove it from here and from ATArchiver.

Comment (srp3rd) [reply sal_interfaces/CCArchiver/CCArchiver_Telemetry.xml]: I just spoke to Htut-Khine, and he said the "sequenceHeartbeat" was an artifact of something called the "Sequencer", which apparently doesn't exist anymore?  I can remove that from the CCArchiver and ATArchiver completely.

**Pull request merged**